docker/README.md

```diff
@@ -0,0 +1,20 @@
+# Docker support
+
+The files in this directory allow you to build a docker image.  The query coordinator assembly must be 
+copied to `query-coordinator-assembly.jar` in this directory before building.
```

Comment:
should we be consistent with jenkins convention and name this `query-coordinator-assembly-LATEST.jar` ?